JSDemos/Demos/List/ListSelection/React/App.js

```diff
@@ -14,82 +14,66 @@ const dataSource = new ArrayStore({
 const selectionModes = ['none', 'single', 'multiple', 'all'];
 const selectAllModes = ['page', 'allPages'];
 
-class App extends React.Component {
-  constructor() {
-    super();
-    this.state = {
-      selectionMode: 'all',
-      selectAllMode: 'page',
-      selectedItemKeys: [],
-    };
-    this.onSelectionModeChange = this.onSelectionModeChange.bind(this);
-    this.onSelectAllModeChange = this.onSelectAllModeChange.bind(this);
-    this.onSelectedItemKeysChange = this.onSelectedItemKeysChange.bind(this);
-  }
+export default function App() {
+  const [selectionMode, setSelectionMode] = React.useState('all');
```

Comment:
We can import useState and useCallback in the top of the file and write useState('all') instead of React.useState('all'), for example

Reply:
This way doesn't work in our demos environment. I think it relates with using of System.js. 

 If we don't write React.useState or React.useCallback this way 
```
import React, { useState, useCallback } from 'react';
...
const [selectionMode, setSelectionMode] = useState('all');
```
we will get the error `TypeError: react_1.useState is not a function`.
Also there is no one demo where this approach is used.

Reply:
@Krijovnick

Reply:
Ok, let's leave it as it is